src/openforms/submissions/tests/test_models.py

```diff
@@ -617,3 +620,20 @@ def test_clear_execution_state_without_execution_state(self):
         self.assertFalse(hasattr(submission, "_execution_state"))
 
         submission.clear_execution_state()
+
+
+@temp_private_root()
+class PDFSubmissionReportTests(HypothesisTestCase):
+    @given(
+        st.text(
+            min_size=1,
+            alphabet=st.characters(blacklist_characters="\x00", codec="utf-8"),
+        )
+    )
+    # see ticket #3470
+    @settings(deadline=500)
+    def test_names_donot_break_pdf_generation(self, form_name):
```

Comment:
```suggestion
    def test_names_do_not_break_pdf_saving_to_disk(self, form_name):
```
slightly better name to address the problem - the PDF generation itself works, but saving it to disk fails